Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -10,6 +10,7 @@
 using AmazonGameLiftPlugin.Core.BucketManagement.Models;
```

Comment:
I'm not sure I understand what this PR is intended for. The description reads

```
 Fixing Bootstrap for added tests.
```

But I don't see any test related changes. Could you help add some more detail to the PR description

Reply:
There is 1 more file I think you missed which you need to load with the "Load Diff" button at the bottom of the files changed section.

Reply:
Also a helpful tip if you like vscode. If you hit fullstop on your keyboard in this menu. The webpage will switch to github dev which could help with file changes.